From: Sagi Grimberg <sagi@grimberg.me>
Date: Sun, 2 Jul 2017 11:20:51 +0300
Subject: nvme-rdma: remove redundant empty device add callout
Patch-mainline: v4.14-rc1
Git-commit: 5138e4bd0bcd1219ccfaaab5458fddc7bca09bfa
References: FATE#323952, FATE#322506

Now that its not needed, we can simply not assign it.

Reviewed-by: Johannes Thumshirn <jthumshirn@suse.de>
Reviewed-by: Christoph Hellwig <hch@lst.de>
Signed-off-by: Sagi Grimberg <sagi@grimberg.me>
Signed-off-by: Doug Ledford <dledford@redhat.com>
---
 drivers/nvme/host/rdma.c |    5 -----
 1 file changed, 5 deletions(-)

--- a/drivers/nvme/host/rdma.c
+++ b/drivers/nvme/host/rdma.c
@@ -1965,10 +1965,6 @@ static struct nvmf_transport_ops nvme_rd
 	.create_ctrl	= nvme_rdma_create_ctrl,
 };
 
-static void nvme_rdma_add_one(struct ib_device *ib_device)
-{
-}
-
 static void nvme_rdma_remove_one(struct ib_device *ib_device, void *client_data)
 {
 	struct nvme_rdma_ctrl *ctrl;
@@ -1990,7 +1986,6 @@ static void nvme_rdma_remove_one(struct
 
 static struct ib_client nvme_rdma_ib_client = {
 	.name   = "nvme_rdma",
-	.add = nvme_rdma_add_one,
 	.remove = nvme_rdma_remove_one
 };
 
